badge-maker/lib/make-badge.spec.mjs

```diff
@@ -709,4 +709,46 @@ describe('The badge generator', function () {
       })
     })
   })
+
+  describe('badges with logo-only should always produce the same badge', function () {
+    it('flat badge, logo-only', async function () {
+      await expectBadgeToMatchSnapshot({
+        label: '',
+        message: '',
+        format: 'svg',
+        logo: 'data:image/svg+xml;base64,PHN2ZyB4bWxu',
+        style: 'flat',
+      })
+    })
+
+    it('flat-square badge, logo-only', async function () {
+      await expectBadgeToMatchSnapshot({
+        label: '',
+        message: '',
+        format: 'svg',
+        logo: 'data:image/svg+xml;base64,PHN2ZyB4bWxu',
+        style: 'flat-square',
+      })
+    })
+
+    it('social badge, logo-only', async function () {
+      await expectBadgeToMatchSnapshot({
+        label: '',
+        message: '',
+        format: 'svg',
+        logo: 'data:image/svg+xml;base64,PHN2ZyB4bWxu',
+        style: 'for-the-badge',
+      })
+    })
+
+    it('plastic badge, logo-only', async function () {
+      await expectBadgeToMatchSnapshot({
+        label: '',
+        message: '',
+        format: 'svg',
+        logo: 'data:image/svg+xml;base64,PHN2ZyB4bWxu',
+        style: 'for-the-badge',
```

Comment:
The `style` values in the test data and the descriptions of the tests here don't match up.

Rather than go through another round of back and forth on this I'm just going to push another commit fixing it so we can get done with this one.

Reply:
Thanks! I've fixed those problems. And sorry about that, I should check it carefully before I submit the code.

> I don't have permission to push to your branch.

That's strange, I haven't set any restrictions on this.